lib/reconcile/rabbitmq.go

```diff
@@ -56,7 +54,12 @@ func EnsureRabbitMQ(cr *apiv1alpha1.Astarte, c client.Client, scheme *runtime.Sc
 	}
 
 	// Depending on the situation, we need to take action on the credentials.
-	if err := handleRabbitMQUserCredentialsSecret(statefulSetName, cr, c, scheme); err != nil {
+	username := cr.Spec.RabbitMQ.Connection.Username
```

Comment:
Maybe avoid all these assignments? To me, secretName is the only one making sense, the rest can simply go in the method invocation. Add a newline if it becomes unreadable.